pkg/oc/oc.go

```diff
@@ -102,3 +102,7 @@ func CheckProjectExists(projectName string) bool{
 	}
 	return true
 }
+
+func CreateSecret(secretData string, namespace string) {
```

Comment:
This function is parametrized but still is not generally usable. There are two options
1. Make it very specific for git resolvers - rename the function as take as few parameters as possible
2. make it very generic - keep name of function, introduce params for secret name and key name